From c22225be9ea5dfce085d73e0fb658d2fea08f859 Mon Sep 17 00:00:00 2001
From: Rosen Penev <rosenp@gmail.com>
Date: Thu, 25 Feb 2021 19:47:00 -0800
Subject: [PATCH] vim: fix compile with BUILD_NLS

Avoids having to use msgfmt, which happens to be broken with gettext-full.

Signed-off-by: Rosen Penev <rosenp@gmail.com>
---
 utils/vim/Makefile                    |  2 +-
 utils/vim/patches/010-no-msgfmt.patch | 11 +++++++++++
 2 files changed, 12 insertions(+), 1 deletion(-)
 create mode 100644 utils/vim/patches/010-no-msgfmt.patch

diff --git a/utils/vim/Makefile b/utils/vim/Makefile
index adc3751a7f4..da2e376304a 100644
--- a/utils/vim/Makefile
+++ b/utils/vim/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vim
 PKG_VERSION:=8.2
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 VIMVER:=82
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
diff --git a/utils/vim/patches/010-no-msgfmt.patch b/utils/vim/patches/010-no-msgfmt.patch
new file mode 100644
index 00000000000..82e19bbc88d
--- /dev/null
+++ b/utils/vim/patches/010-no-msgfmt.patch
@@ -0,0 +1,11 @@
+--- a/src/po/Makefile
++++ b/src/po/Makefile
+@@ -32,7 +32,7 @@ MSGMERGE = OLD_PO_FILE_INPUT=yes OLD_PO_
+ 	$(VIM) -u NONE -e -X -S check.vim -c "if error == 0 | q | endif" -c cq $<
+ 	touch $@
+ 
+-all: $(MOFILES) $(MOCONVERTED) $(MSGFMT_DESKTOP)
++all: $(MOFILES) $(MOCONVERTED)
+ 
+ check: $(CHECKFILES)
+ 
